src/pages/dar_application/ResearcherInfo.js

```diff
@@ -145,7 +141,6 @@ export default function ResearcherInfo(props) {
           div({ className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group' }, [
             label({ className: 'control-label rp-title-question' }, [
               '1.3 Principal Investigator* ',
-              //NOTE: user doesn't actually type in the name, it's prefilled. Should I adjust phrasing?
```

Comment:
I did miss this question the first time around, sorry. Yes, if you want to adjust the phrasing, that would be a good idea. Perhaps something like 
```
"I certify that the principal investigator is aware of this study."
```
@jlawsonbroad Do you have a better suggestion?

Reply:
"I certify that the principal investigator listed below is aware of this study."

Thanks @JVThomas and @rushtong